standalone/README.md

```diff
@@ -47,11 +47,14 @@ The `standalone.py` script includes a main command to execute the full workflow,
 subcommands to run individual parts of the workflow separately. The full workflow includes fetching SDG data from S3, training,
 and evaluating a model. To view all available commands, use `./standalone.py --help`.
 
-The script requires information regarding the location and method for accessing the SDG data. This information can be provided in two main ways:
+The script requires information regarding the location and method for accessing the SDG
+data/model/taxonomy tree and the evaluation Judge model serving endpoint. This information can be
+provided in two main ways:
 
 1. CLI Options or/and Environment Variables: Supply all necessary information via CLI options or environment variables.
-    * See [CLI Options](#cli-options) for full list.
-2. Kubernetes Secret: Provide the name of a Kubernetes secret that contains all relevant details using the `--sdg-object-store-secret` option.
+    * See [CLI Options](#cli-options) for full list. In particular `--sdg-object-store-*` and `--judge-serving-model-*` options.
+2. Kubernetes Secret: Provide the name of a Kubernetes secret that contains all relevant details
+   using the `--sdg-object-store-secret` option and `--judge-serving-model-secret` option.
     *  See [Creating the Kubernetes Secret for S3 Details](#creating-the-kubernetes-secret-for-s3-details) for information on how to create the secret.
 
 The examples below assume there is a secret in `my-namespace` named `sdg-data` that holds information about the S3 bucket.
```

Comment:
also update #L60-62, something like:

The examples below assume there are the following secrets in `my-namespace`: `sdg-data` secret that includes information about the S3 bucket and `judge-serving-details` secret that includes information about the judge server model. A judge server model is assumed to run external to the script. See [Judge Model Details](#judge-model-details) for required information.

Reply:
Addressed thanks.